source/documentation/services/domain-naming-standard.html.md.erb

```diff
@@ -164,4 +164,27 @@ The Operations Engineering team will then:
 - work with you to register your domain
 - manage renewals for the domain
 - set up the domain to adhere to the [MOJ's Security Guidance]
-  (https://security-guidance.service.justice.gov.uk/#cyber-and-technical-security-guidance)
+  (<https://security-guidance.service.justice.gov.uk/#cyber-and-technical-security-guidance>)
+
+## Assessing Existing Domains
+
+If you already have a domain that you are using to provide Ministry of Justice Services and wondering if it is compliant with our standards, you can follow these steps assess the domain:
+
+- ✅ If the domain name ends with `gov.uk`, it is compliant with our standards. All domains with this Top Level Domain will have gone through the relevant checks and approvals before being created.
+- ✅ If the domain name does not end with `gov.uk`, [check if the domain is owned by the Ministry of Justice](#check-if-a-domain-is-owned-by-the-ministry-of-justice). After, [check if the domain is deprecated](#check-if-a-domain-is-deprecated). If the domain is owned by the Ministry of Justice and is not deprecated, then it is probably compliant with our standards. It still may be worth considering migrating the service to the `gov.uk` domain where appropriate - but this is not a requirement in all cases.
+
+### Check if a Domain is Owned by The Ministry of Justice
+
+You can check if the domain is owned by the Ministry of Justice by looking up the domain name in our public [DNS Repository](https://github.com/ministryofjustice/dns/blob/main/hostedzones). If the Second Level Domain is owned by the Ministry of Justice, there will be Hosted Zone file for the domain in the repository. If there is not a Hosted Zone file for the domain, it is not owned by the Ministry of Justice.
```

Comment:
Not all Hostedzones that we own are in that repo. Might be worth saying that if it is not in DNS, that the user check with domains@digital.justice.gov.uk. Manual step but many questions would be answered by checking DNS repo in the first instance.